### DISCUSSION TITLE
Swell Checkout

### DISCUSSION THREAD

**Discussion by Daniel-Alamezie** [swellstores/community#433] [Help] [answered]
So i know swell has it own checkout that looks something like this
https://exoplate-store.swell.store/checkout/test/.../shiping
now I have been speaking to someone on here who mentioned that this solely for test purposes only but said I can add a different payment gateway and when a customer checks out using the https://exoplate-store.swell.store/checkout/test/ the payment gets refelected on the payment gateway (e.g stripe).

Now i have added Stripe as a payment gateway to my swell dashboard and i have made some test payments and i cant see these test payments show up on stripe.

Can someone help me with this, am i missing something here?

**Comment by sags95** [comment]
Hey @Daniel-Alamezie, I had a chance to take a look at your store. It looks like Stripe is enabled for live environment, but not for test environment. On the Swell Dashboard, if you switch to "test" by clicking the "live" section at the top left and switch the toggle, you can then connect the Swell test environment with Stripe test environment. After this you should be able to see orders in test mode flow through to Stripe test env.

**Comment by Daniel-Alamezie** [comment]
Ahh i see this begs the next question. Since i have been working on test mode all this time, when i switch to live, do i need to generate new keys and use live API keys to access swell?
@sags95

**Reply by sags95** [reply, marked as answer]
Yes Swell environments have a separate set of public keys and secret keys (the data in each env is different as well). So you'll have to configure your setup with the live mode keys before going live.
